specification/network/resource-manager/Microsoft.Network/stable/2018-12-01/webapplicationfirewall.json

```diff
@@ -0,0 +1,549 @@
+          "enum": [
+            "MatchRule",
+            "Invalid"
+          ],
+          "x-ms-enum": {
+            "name": "WebApplicationFirewallRuleType",
+            "modelAsString": true
+          }
+        },
+        "matchConditions": {
+          "description": "List of match conditions",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/MatchCondition"
+          }
+        },
+        "action": {
+          "description": "Type of Actions",
+          "type": "string",
+          "enum": [
+            "Allow",
+            "Block",
+            "Log"
+          ],
+          "x-ms-enum": {
+            "name": "WebApplicationFirewallAction",
+            "modelAsString": true
+          }
+        }
+      }
+    },
+    "Transform": {
+      "description": "Describes what transforms applied before matching",
+      "type": "string",
+      "enum": [
+        "Lowercase",
+        "Trim",
+        "UrlDecode",
+        "UrlEncode",
+        "RemoveNulls",
+        "HtmlEntityDecode"
+      ],
+      "x-ms-enum": {
+        "name": "WebApplicationFirewallTransform",
+        "modelAsString": true
+      }
+    },
+    "MatchVariable": {
+      "description": "Define match variables",
+      "required": [
+        "variableName"
+      ],
+      "properties": {
+        "variableName": {
+          "description": "Match Variable",
+          "type": "string",
+          "enum": [
+            "RemoteAddr",
+            "RequestMethod",
+            "QueryString",
+            "PostArgs",
+            "RequestUri",
+            "RequestHeaders",
+            "RequestBody",
+            "RequestCookies"
+          ],
+          "x-ms-enum": {
+            "name": "WebApplicationFirewallMatchVariable",
+            "modelAsString": true
+          }
+        },
+        "selector": {
+          "description": "Describes field of the matchVariable collection",
+          "type": "string"
+        }
+      }
+    },
+    "MatchCondition": {
+      "description": "Define match conditions",
+      "required": [
+        "matchVariables",
+        "operator",
+        "matchValues"
+      ],
+      "properties": {
+        "matchVariables": {
+          "description": "List of match variables",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/MatchVariable"
+          }
+        },
+        "operator": {
+          "description": "Describes operator to be matched",
+          "type": "string",
+          "enum": [
+            "IPMatch",
+            "Equal",
+            "Contains",
+            "LessThan",
+            "GreaterThan",
+            "LessThanOrEqual",
+            "GreaterThanOrEqual",
+            "BeginsWith",
+            "EndsWith",
+            "Regex"
+          ],
+          "x-ms-enum": {
+            "name": "WebApplicationFirewallOperator",
+            "modelAsString": true
+          }
+        },
+        "negationConditon": {
```

Comment:
Conditon -> Condition

Reply:
Server-side typoe, nothing we can do
https://github.com/Azure/azure-rest-api-specs/pull/5341#discussion_r263960243

Reply:
I guess the next API version would fix it then?

Reply:
That's a question for @azhao155